Org migration (#397)
bitwarden/help: _articles/importing/teams-enterprise-migration-guide.md

---
layout: article
title:  Teams and Enterprise Organization Migration Guide
categories: [import-export]
featured: false
popular: false
hidden: false
tags: [import, teams, enterprise, onboarding, security]

> order: 98

Reviewer: No need to bury this 🔥. We can set this to `order: 09`, and shift /import-faqs/ to `order: 10`.

Author: No worries, we can move around as needed 👍

---
## Bitwarden: Making migration easy

Securing your Organization with Bitwarden is straightforward and secure.

Simply follow the below steps to migrate data and users from your existing password manager:

1. [Export your data](#exporting-data)
2. [Create and configure your Bitwarden Organization](#creating-an-organization)
3. [Import your data into Bitwarden](#importing-data)
4. [Onboard your users](#onboarding-users) and [share](#sharing-collections-and-items)

If you need assistance during your migration, our [Customer Success team is here to help!](#migration-support)

## Scope

> This document will describe the best practices for migrating secure data from your current password manage(s) to a Bitwarden Teams or Enterprise Organization, building an infrastructure for security based on simple and scalable methods.

Reviewer: ```suggestion
This document will describe the best practices for migrating secure data from your current password manage(s) to a Bitwarden Teams or Enterprise Organization, building an infrastructure for security based on simple and scalable methods.
```
```suggestion
This document will describe the best practices for migrating secure data from your current password manager(s) to a Bitwarden Teams or Enterprise Organization, building an infrastructure for security based on simple and scalable methods.
```

> This document will describe the best practices for migrating secure data from your current password manage(s) to a Bitwarden Teams or Enterprise Organization, building an infrastructure for security based on simple and scalable methods.

Reviewer: ```suggestion
This document will describe the best practices for migrating secure data from your current password manage(s) to a Bitwarden Teams or Enterprise Organization, building an infrastructure for security based on simple and scalable methods.
```
```suggestion
This document will describe the best practices for migrating secure data from your current password manager(s) to a Bitwarden Teams or Enterprise Organization, building an infrastructure for security based on simple and scalable methods.
```

> This document will describe the best practices for migrating secure data from your current password manage(s) to a Bitwarden Teams or Enterprise Organization, building an infrastructure for security based on simple and scalable methods.

Reviewer: ```suggestion
This document will describe the best practices for migrating secure data from your current password manager(s) to a Bitwarden Teams or Enterprise Organization, building an infrastructure for security based on simple and scalable methods.
```

Password management is crucial for organizational security and operational efficiency. Providing insight into the best methods to perform migration and configuration is designed to minimize the trial-and-error approach that is often needed when exchanging enterprise tools.

> The below steps are listed in the recommended order for ease of use and smooth boarding for users.

Reviewer: ```suggestion
The below steps are listed in the recommended order for ease of use and smooth onboarding for users.
```

## Importing data

Importing information into Bitwarden can be performed in both an Individual Vault and Organization Vault. The below instructions are for Organization Imports.

There are two options to import data into your Organization:

1. Using the default CSV export file from your prior password manager
2. Creating a Bitwarden specific CSV from your exported data

The best practice for most Organizations is to format your data into a Bitwarden CSV, or for advanced users, a Bitwarden JSON file for import into your Organization vault.

For instructions on shaping a Bitwarden specific import file, please refer to the guide [here.](https://bitwarden.com/help/article/condition-bitwarden-import/)

 A collection of data import and export documentation is available [here.](https://bitwarden.com/help/import-export/) to assist with imports from additional sources.

{%callout%}
Importing multiple times will create duplicate records in your Bitwarden Vault.
{%endcallout%}

### Individual user data import

> Bitwarden supports a variety import formats from other password management platforms. Individual users can import their data into their Bitwarden Vault on their own, and does not require Administrative assistance.

Reviewer: ```suggestion
Bitwarden supports a variety of import formats from other password management platforms. Individual users can import their data into their Bitwarden Vault on their own, and do not require Administrative assistance.
```

For more on importing individual data, check out our helpful article [here.](https://bitwarden.com/help/import-export/)

## Onboarding users

> Bitwarden supports both manual and automated user invitation and boarding. Best practice is to manually board any necessary Administrative users during configuration and initial deployment, ensuring all Organization configurations including Enterprise Policies, Login with SSO, and Directory Connector are ready prior to automating the User invitation and boarding process.

Reviewer: ```suggestion
Bitwarden supports both manual and automated user invitation and onboarding. Best practice is to manually onboard any necessary Administrative users during configuration and initial deployment, ensuring all Organization configurations including Enterprise Policies, Login with SSO, and Directory Connector are ready prior to automating the User invitation and onboarding process.
```

### Manual onboarding

> Manual boarding is done via the Web Vault. More information on manual user boarding can be found in this [helpful article](https://bitwarden.com/help/article/managing-users/)

Reviewer: ```suggestion
Manual onboarding is done via the Web Vault. More information on manual user onboarding can be found in this [helpful article](https://bitwarden.com/help/article/managing-users/)
```

### Automated onboarding

> Automated user boarding is also available when leveraging Bitwarden Directory Connector - a standalone application available in a [Desktop app](https://bitwarden.com/help/article/directory-sync-desktop/) and a [CLI tool](https://bitwarden.com/help/article/directory-sync-cli/) -  synchronizing user and group information to the Bitwarden Organization. These users are automatically invited to join the Organization, and can be confirmed manually or automatically using the [Bitwarden CLI tool](https://bitwarden.com/help/article/cli/#confirm).

Reviewer: ```suggestion
Automated user onboarding is also available when leveraging Bitwarden Directory Connector - a standalone application available in a [Desktop app](https://bitwarden.com/help/article/directory-sync-desktop/) and a [CLI tool](https://bitwarden.com/help/article/directory-sync-cli/) -  synchronizing user and group information to the Bitwarden Organization. These users are automatically invited to join the Organization, and can be confirmed manually or automatically using the [Bitwarden CLI tool](https://bitwarden.com/help/article/cli/#confirm).
```
